_posts/2019-07-15-hccamp.md

```diff
@@ -0,0 +1,82 @@
+
+(missing some text here)
+
+The conference itself consisted of 2 different tracks: Campfire and Compass.
+
+### Campfire Track
+
+Following an open space format, both conference days started with a welcoming speech and a Marketplace. Every camper introduced to the others one or more topics that he/she was interested about and placed it in the schedule wall, including 5 different spaces and 7 time slots per day.
+
+There are some basic rules to keep in mind for an open space:
+
+- Whoever comes is the right people
+- Whatever happens is the only thing that could’ve
+- When it starts is the right time
+- When it’s over, it’s over 
+
+We had more than 60 different topics to discuss about, so exciting as:
+
+- how to make our product **faster** and more **performant**
+- how to bring **sustainability** to the travel industry
+- how to improve the **transparency** between management and teams
+- how could we improve the **collaboration** between UX and developers
+- what is the future of **mobile and cross-device development** at HolidayCheck
+- how to do **urlauber-friendly ads**
+- we did some **hacking** and **coding** sessions
+- and much more: **testing, innovation, deep learning, data and services, visual development or meeting culture**
+- and we even had the chance to learn some **salsa** and **charleston** moves or get to know some insides about the **Indian culture** from our Indian colleagues
+
+### Compass track
+
+New this year we did a *classic* conference track where our great HolidayCheck people could bring out the speaker in them and present us the topic they are passionate about. 
+
+Some weeks before the HC.Camp we opened a Call for Papers that got more than 20 amazing proposals that formed a full packed track of 7 talks for the second day of the conference. Also all-new this year, the talks were being live streamed to our offices and recorded. If you want to see our amazing speakers (again) on stage, these are the links to the videos:
+
+- Georg Köhler // [Relationship innovation, should we care?](https://youtu.be/--Ct4qdShM4)
+- Francesco Zanini // [Farewell Docker](https://youtu.be/x-SccVIlbl0)
+- Marta Kosswoska // [How to be a good micro-manager](https://youtu.be/8-MovYbK3Hg)
+- Vladimir Plokhov // [The formula of love or "True power of quantification and decomposition"](https://youtu.be/q1zKaSfera0)
+- Branko Dzakula // [D-Day: Responding to a data breach in the right way](https://youtu.be/Hm6RKZiNJmA)
+- Gabriel Bernal // [Augmented and Virtual Reality in the travel industry](https://youtu.be/6ptXi3Pu3aY)
+- Artur Szott // [Where do good ideas come from?](https://youtu.be/A0exESwnC7Q)
+
+## What now
+
+After the camp is only before the camp. We are so much looking forward to the next edition and we are excited about every new idea that comes to the table. Stay tuned!
```

Comment:
`After the camp is only before the camp.` -> `The camp is over, long live the camp!`

Reply:
`Stay tuned!` -> `In a few months we will start sharing more details, we have your curiosity and we will try to have your attention!`